lib/utils/autocomplete.js

```diff
@@ -0,0 +1,107 @@
+'use strict';
+
+const tab = require('tabtab')({
+  name: 'serverless',
+});
+
+const autocomplete = () => {
+  tab.on('serverless', (data, done) => {
+    done(null, [
+      'create',
+      'install',
+      'pacakge',
+      'deploy',
+      'invoke',
+      'logs',
+      'login',
+      'metrics',
+      'info',
+      'rollback',
+    ]);
```

Comment:
Agreed, and I also feel like all the 'on' below could be done by grammatically examining their options?